xml/deployment_cluster_administration.xml

```diff
@@ -106,26 +106,26 @@ mail: <email address>
      ]]>
     </screen>
     <para>
-     To generate the password hash, install the <package>openldap2</package>
-     package:
-    </para>
-    <screen>&prompt.root; <command>zypper</command> <option>install openldap2</option></screen>
-    <para>
-     Then generate the password hash:
+     To generate the password hash, on a separate machine, install the 
+     <package>openldap2</package> package:
     </para>
+    <screen>
+     &prompt.root; <command>zypper</command> <option>install openldap2</option>
```

Comment:
Make screen lines flush left, unless you want leading spaces

Reply:
Done.